security/index.html

```diff
@@ -1,82 +1,14 @@
 ---
 layout: news
-title: Security Vulnerabilities
+title: Security
 filename: security
-main-blurb: Known vulnerabilities, workarounds, and resolutions
```

Comment:
We probably want a new blurb here
"Keeping your data secure" ?

Reply:
Probably. I can push `Keeping your data secure` and leave others to open follow-up PRs here and in the following section.